client/src/app/create/create.component.ts

```diff
@@ -24,7 +24,7 @@ export class CreateComponent implements OnInit {
 
     this.wordListService.wordList$
       .subscribe(results => {
-        const wordList = results.slice().reverse();
+        const wordList = [...results].reverse();
         this.wordList = wordList.map(item => Object.assign({}, item, {edit: false}));
```

Comment:
nie mogłem zaimplementować operatora rozwijania dla obiektu
`wordList.map(item =>{ ...item, edit: false}));`
można, by zaciągnąć [transpiler](https://babeljs.io/docs/plugins/transform-object-rest-spread/), ale nie ma sensu, jeśli to nie jest wbudowane standardowo. Co myślisz?